packages/@okta/vuepress-site/docs/guides/terraform-create-custom-auth-server/main/index.md

```diff
@@ -0,0 +1,396 @@
+---
+title: Create a custom authorization server
+excerpt: Configure single sign-on and control resource permissions for an OAuth app.
+layout: Guides
+---
+
+Configure single sign-on and control resource permissions for an OAuth app.
+
+---
+
+#### Learning outcomes
+
+* Why use an authorization server.
+
+* When to use an org authorization server versus a custom authorization server.
+
+* Create an authorization server using Terraform.
+
+#### What you need
+
+* Familiarity with the Terraform terms: configuration, resources, and commands. See the Terraform documentation [introductory concepts](https://www.terraform-best-practices.com/key-concepts) article. Also, see the [introduction page for Okta Terraform automation](/docs/guides/terraform-overview/main/).
+
+* An Okta org and pricing plan that includes support for authorization servers. [Okta's Developer Edition](https://developer.okta.com/signup/) does not include [API Access Management](/docs/concepts/api-access-management/), which is required to use custom authorization servers.
+
+* A [Terraform configuration](/docs/guides/terraform-enable-org-access/main) that can access your Okta org.
+
+* Admin permissions that enable you to add a scope to your org.
+
+* Understand your OAuth app's needs for proving identity and resource access.
+
+* Familiarity with concepts and terminology of OAuth and OIDC, such as audience, scopes, claims, and grants. See [OAuth 2.0 and OpenID Connect overview](/docs/concepts/oauth-openid/).
+
+---
+
+## Overview
+
+A web or mobile app must handle authentication (user identity) and authorization (resource access for each user). Whether you're developing an internal IT app for your employees, building a portal for your partners, or exposing custom APIs to users, you can use Okta to manage authentication and authorization.
+
+OAuth 2.0 and OpenID Connect (OIDC) are the industry standard user authentication and authorization protocols. See [OAuth 2.0 and OpenID Connect overview](/docs/concepts/oauth-openid/). Okta supports both these protocols by checking user access and generating appropriate tokens.
```

Comment:
```suggestion
OAuth 2.0 and OIDC are the industry standard user authentication and authorization protocols. See [OAuth 2.0 and OpenID Connect overview](/docs/concepts/oauth-openid/). Okta supports both of these protocols by checking user access and generating appropriate tokens.
```

Reply:
or "both protocols"

Reply:
fixed in next commit (using shorter alternate suggestion)